tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -52,6 +52,32 @@ module.exports = {
       return this.waitForElementPresent('@messageCloseIcon')
         .click('@messageCloseIcon')
         .waitForElementNotPresent('@messageCloseIcon')
+    },
+    getNotifications: async function () {
+      const notifications = []
+      await this
+        .click('@notificationBell')
+        .waitForElementVisible('@notificationElement')
+      await this.api.elements('@notificationElement', async result => {
+        result.value.map(item => {
+          notifications.push(new Promise((resolve) => {
```

Comment:
Any reasons why this is promisified? 

You could just await and `push` to the array and return all of them later on. I feel that'd be simpler.